src/containers/app/app.jsx

```diff
@@ -101,7 +150,14 @@ const AppContainer = ({ route }) => (
             <main>
                 {renderRoutes(route.routes)}
             </main>
-            <Footer>Footer is supposed to be here</Footer>
+            <Footer>
```

Comment:
Maybe we should move it to separate component? I thing it is too much code here.